libpod/networking_linux.go

```diff
@@ -301,206 +302,218 @@ func (r *RootlessCNI) Cleanup(runtime *Runtime) error {
 
 // GetRootlessCNINetNs returns the rootless cni object. If create is set to true
 // the rootless cni namespace will be created if it does not exists already.
+// If called as root it returns always nil.
+// On success the returned RootlessCNI lock is locked and must be unlocked by the caller.
 func (r *Runtime) GetRootlessCNINetNs(new bool) (*RootlessCNI, error) {
+	if !rootless.IsRootless() {
+		return nil, nil
+	}
 	var rootlessCNINS *RootlessCNI
-	if rootless.IsRootless() {
-		runDir, err := util.GetRuntimeDir()
-		if err != nil {
-			return nil, err
-		}
-		cniDir := filepath.Join(runDir, "rootless-cni")
-		err = os.MkdirAll(cniDir, 0700)
-		if err != nil {
-			return nil, errors.Wrap(err, "could not create rootless-cni directory")
-		}
-
-		lfile := filepath.Join(cniDir, "rootless-cni.lck")
-		lock, err := lockfile.GetLockfile(lfile)
-		if err != nil {
-			return nil, errors.Wrap(err, "failed to get rootless-cni lockfile")
-		}
-		lock.Lock()
-		defer lock.Unlock()
+	runDir, err := util.GetRuntimeDir()
+	if err != nil {
+		return nil, err
+	}
 
-		nsDir, err := netns.GetNSRunDir()
-		if err != nil {
-			return nil, err
+	lfile := filepath.Join(runDir, "rootless-cni.lck")
```

Comment:
```suggestion
	lfile := filepath.Join(runDir, "rootless-cni.lock")
```

to be consistent with other lock files.

Reply:
done